src/xcode/ENA/ENA/Source/Services/CCLService/CCLService.swift

```diff
@@ -61,7 +61,7 @@ class CLLService: CCLServable {
 	}
 	
 	func evaluateFunction<T>(name: String, parameters: [String: AnyDecodable]) throws -> T where T: Decodable {
-		return try JsonFunctions().evaluateFunction(name: name, parameters: parameters)
+		return try jsonFunctions.evaluateFunction(name: name, parameters: parameters)
```

Comment:
I just realized this is specific for the UI texts, and maybe we should have a separate func for that, but according to the [tech spec](https://github.com/corona-warn-app/cwa-app-tech-spec/blob/proposal/logic-engine/docs/spec/dcc-ccl.gen.md#formatting-a-ui-text---system-time-dependent) we need to pass in the default parameters (os, language, now) for these in addition to the parameters that are passed in. 

Maybe we can just extend this implementation to always pass in the default parameters as well, because currently we never need a function call without them.

Reply:
Good point. I have changed the func name and added the default parameters.